projects/packages/masterbar/src/admin-menu/admin-menu.css

```diff
@@ -30,13 +30,15 @@
  * Fixes Gutenberg in not fullscreen mode.
  */
  @media (min-width: 783px) {
+
```

Comment:
I found there are lots newline at the beginning of the media section, it is intended?

Reply:
No, it looks like something in my IDE is doing that, not sure what yet.